data/git-datasets.md

```diff
@@ -147,20 +134,15 @@ If you have already cloned a repository and would like to get the latest version
 
 ## Getting write access to a repository
 
-By default, all lab members have _read_ access to all repositories, so you can [clone repositories](#connecting-from-the-command-line-cloning-a-repo) and [pull changes from the server](#getting-changes-from-the-server). If you want to [open issues](#opening-an-issue) or [push changes to the server](#making-a-pull-request), you will need _write_ access to the repository in question.
+By default, all lab members have _read_ access to all repositories, so you can [clone repositories](#connecting-from-the-command-line-cloning-a-repo) and [pull changes from the server](#getting-changes-from-the-server). If you want to [push changes to the server](#making-a-pull-request), you will need _write_ access to the repository in question.
 
 To get write access to a repository, ask [one of the current admins](#getting-help). They will need to use the web interface, go to the repository's "Settings" tab, and add your username to the list of "Collaborators".
 
 ## Opening an issue
 
 If you're working with a dataset, and you notice that something is wrong, or something could be better, consider opening an issue!
 
-1. If it's your first issue or pull request for the repository, [ask for write access](#getting-write-access-to-a-repository).
-2. In the web interface, go to the repository page.
-3. Click on the "Issues" tab.
-4. Click "New Issue".
-5. Fill in the title and describe the issue in as little or as much detail as you want.
-6. Click "Create Issue".
+Since we sometimes want to discuss issues with people outside the lab, we use [Github issues in `neuropoly/data-management`](https://github.com/neuropoly/data-management/issues), which is publically visible to the whole internet. Open an issue there, and avoid putting private information there (but you can include links to <https://data.neuro.polymtl.ca/>).
```

Comment:
Typo
```suggestion
Since we sometimes want to discuss issues with people outside the lab, we use [Github issues in `neuropoly/data-management`](https://github.com/neuropoly/data-management/issues), which is publicly visible to the whole internet. Open an issue there, and avoid putting private information there (but you can include links to <https://data.neuro.polymtl.ca/>).
```